app/src/main/java/org/wikipedia/page/PageFragmentLoadState.java

```diff
@@ -186,6 +186,24 @@ private void pageLoadFromNetwork(final ErrorCallback errorCallback) {
 
         app.getSessionFunnel().leadSectionFetchStart();
 
+        model.setPage(null);
+        boolean delayLoadHtml = model.getTitle().getPrefixedText().contains(":");
+
+        if (!delayLoadHtml) {
+            bridge.resetHtml(model.getTitle());
+        }
+
+        if (model.getTitle().namespace() == Namespace.SPECIAL) {
+            // Short-circuit the entire process of fetching the Summary, since Special: pages
+            // are not supported in RestBase.
+            model.setPage(null);
```

Comment:
Is it necessary since line `189` has already set it to `null`?

Reply:
nope